esa/esa_test.go

```diff
@@ -0,0 +1,363 @@
+package esa
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"reflect"
+	"strings"
+	"testing"
+	"time"
+)
+
+const (
+	token = "b81f8961eb0a8c53faf69faf71b7d62dab9e581f7d2f5344f7f0d686e9e711f8"
```

Comment:
**[unused]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
const token is unused (U1000)